midcli/command/override/interface.py

```diff
@@ -29,6 +39,16 @@ def patch_interface(context, interfaces):
             for alias in interface["aliases"]
         ]
 
+        interface.pop("fake")
+        interface.pop("state")
+
+        interfaces[i] = {
+            "name": interface["name"],
+            "type": interface["type"],
+            **{k: v for k, v in interface.items() if k.startswith("state.")},
+            **{k: v for k, v in interface.items() if not (k in ["name", "type"] or k.startswith("state."))},
```

Comment:
Are we repeating this to have the dictionary keys in a specific order ?

Reply:
Yes